package.json

```diff
@@ -8,6 +8,7 @@
     "build": "tsc",
     "start": "npm run build && cross-env NODE_ENV=production node dist/",
     "dev": "cross-env NODE_ENV=development nodemon src/",
+    "test": "cross-env NODE_ENV=test jest",
```

Comment:
We will need to pass the testing database url here so that it will override the DATABASE_URL value in the `.env` file. 

```js
    "test": "cross-env NODE_ENV=test DATABASE_URL=mongodb://localhost:27017/test jest --detectOpenHandles",
```